recipe/meta.yaml

```
@@ -1,24 +1,27 @@
 {% set name = "debugpy" %}
-{% set version = "1.8.11" %}
+{% set version = "1.8.16" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6ad2688b69235c43b020e04fecccdf6a96c8943ca9c2fb340b8adc103c655e57
+  sha256: 31e69a1feb1cf6b51efbed3f6c9b0ef03bc46ff050679c4be7ea6d2e23540870
 
 build:
   number: 0
   entry_points:
     - debugpy = debugpy.server.cli:main
```

```suggestion
    - debugpy = debugpy.server.cli:main
    - debugpy-adapter = debugpy.adapter.__main__:main
```
https://github.com/microsoft/debugpy/blob/v1.8.16/setup.py#L202